File: FizzBuzz.java
public class FizzBuzz {
    public static void main(String[]args) {
	for (int i = 0; i < 100; i++) {
	    if (i % 3 == 0 && i % 5 == 0) {
		System.out.println("FizzBuzz");
	    } else if (i % 3 == 0) { 
		System.out.println("Fizz");
	    } else if (i % 5 == 0) { 
		System.out.println("Buzz");
	    } else {
		System.out.println(i);
	    }
	}
    }
}

File: src/main/php/FizzBuzz.php
<?php

for($i=1;$i<=100;$i++){
    if(($i % 15)==0){ print 'FizzBuzz;'."\n<br />"; }
    elseif(($i % 3)==0){ print 'Fizz;'."\n<br />"; }
    elseif(($i % 5)==0){ print 'Buzz;'."\n<br />"; }
    else{ print $i."\n<br />"; }
}
?>
